doc/conf.py

```diff
@@ -59,7 +59,7 @@
     'examples_dirs': '../examples',
     # path where to save gallery generated examples
     'gallery_dirs': 'auto_examples',
-    'filename_pattern': '/\w+'}
+    'filename_pattern': r'/\w+'}
```

Comment:
That changes the semantics, right? also that is the conf of the doc....

Reply:
this string contains backslash character. so, we should use "r" prefix for treat backslashes as literal characters

Reply:
ooohh for some reason I thought it was escaped but it's not... you're right.